src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php

```diff
@@ -524,7 +528,7 @@ private function getCalculatedPrices($type = 'rent', $currency) {
             //   ['priceNettoPerSqmPerYear', 'renders'],
             //   ['priceNettoPerTotalPerMonth', 'renders'],
             //   ['priceNettoPerTotalPerYear', 'renders'],
-            ['extraCosts', 'special'],
```

Comment:
Did you remove this, mecause we are now always using extraCostsPerMonth?

Reply:
@josias-r yes correct otherwise it'll be double and on rent - but yes now it'll be limited to the monthly calculation...
maybe it would make sense to adjust the logic and make it dependable on if monthly or yearly?

Reply:
Hmm I see... Well from the cassone side, the values are always displayed as "monthly" (rental).
I think that's why I added it in the first place, so all values are consistently formatted.

TLDR casaone won't make use of the yearly value, so we don't need it for that sake, idk about other services